components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -199,4 +201,29 @@ public long getTimestamp(String date) {
         }
         return time;
     }
+
+    /**
+     * configure email list which need to send alert
+     * update email list as adding more or remove
```

Comment:
what does this mean?

Reply:
The update email list part needs to remove, I am removing it. because early I created a single input which can add and remove multiple emails. Currently, it has changed can add only. Deleting happens a separate task.